### PR TITLE
Features/add german labels definitions

### PR DESCRIPTION
**Pull request by srhbrnds**
Add German labels to input and output parameter list

**Changes proposed in this pull request**:
- add columns with German translation in output_parameter_list.csv and input_parameter_list.csv and

### CODE REVIEW

File: docs/_files/input_parameters_list.csv [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
+Existing grid structure (v2.0 of the tool),existierende Netzstruktur,GPS Koordinaten der existierenden Netzinfrastruktur,GPS coordinates of current grid structure (lines and hubs),,,,GIS/Technical,,hidden,,,"in the Ego-Tools I guess, but this is a topic I am not familiar with",,
+Future grid structure (v2.0 of the tool),zuk�nftige Netzstruktur,GPS Koordinaten der zuk�nftigen Netzinfrastruktur,GPS coordinates of future grid structure (lines and hubs),,,,GIS/Technical,,hidden,,,"in the Ego-Tools I guess, but this is a topic I am not familiar with",,
+Security of supply,Versorgungssicherheit,"Welche Technologien existieren, um die Versorgungssicherheit zu gew�hrleisten (z.B.  Notstromaggregat)", e.g. existence of back-up system,,,,Technical,,hidden,,,there are many components and constraints for a flexible backup system,,
+Balancing ratio for anciliary services,?,?,Balancing ratio for ancilliary services (need a more precise definition as PF could not re-figure out what this means),,,array,Technical,,hidden,,,?,,
```
Comment (srhbrnds) [inline docs/_files/input_parameters_list.csv:28]: I have no idea what is meant with this parameter_name. Can you help out @Bachibouzouk ?

File: docs/_files/input_parameters_list.csv [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
+Number of future extra users,?,?,Number of future extra user in energy cell,,,int,Technical,,hidden,,,?,,
```
Comment (srhbrnds) [inline docs/_files/input_parameters_list.csv:29]: I have no idea what is meant with this parameter_name. Can you help out @Bachibouzouk ?

File: docs/_files/input_parameters_list.csv [hunk 3]
```diff
@@ -1,32 +1,32 @@
...
+Source power mix,Anteil Energieerzeuger am Energiemix,Anteil jedes Energieerzeugers am Energiemix (inkl. % Netzbezug),Percentage of each source of power (inclusive % from Grid),Percentage,,array,Technical,,constraint,,OP_IN_05,component/constraint exists,,
+Lower bound renewable energy share ,min. EE-Anteil,"Eine Randbedingung, welche den minimalen Anteil EE-Energieerzeuger im Bezug auf den optimierten Energiemix festlegt",A constraint that specifies the lower bound on the share of renewable energies in the optimized output energy mix,Percentage,None,float,Technical,,constraint,,,component/constraint exists,,
+CO2 upper limit,max. CO2-Emissionen,"Eine Randbedingung, welche die maximale Masse an CO2-Emissionn festlegt",A constraint that specifies an upper bound on the CO2 emissions,T of CO2,None,float,Environmental,,constraint,,,component/constraint exists,,
+Energy sources capacities,Erzeugerkapazit�ten,"Erzeugertechnologien, die Entscheidungsvariablen der Optimierung sind mit bereits installierte Kapazit�ten",Energy sources with their capacity open for optimization ,kW,None,float,Technical,,component,,,?,,A user can select
```
Comment (srhbrnds) [inline docs/_files/input_parameters_list.csv]: Looks like an Encoding problem. I edited the csv file using Excel. Shall I change the encoding before saving the file?

Comment (Bachibouzouk) [reply docs/_files/input_parameters_list.csv]: Excel is bad at editing csv which should be reusable/reopenable by other softwares than Excel ... (probably a strategy?) It is better to use LibreOffice to edit csv files with a tabular view ;)

File: docs/_files/input_parameters_list.csv (hunk 1 above)
Comment (Bachibouzouk) [reply docs/_files/input_parameters_list.csv:28]: @FHell how would you translate "Balancing ratio for anciliary services" in german?

File: docs/_files/input_parameters_list.csv (hunk 2 above)
Comment (Bachibouzouk) [reply docs/_files/input_parameters_list.csv:29]: It was a parameter I had in mind to be able to build an energy cell from a standard indivudual load profiles multiplied by the number of users

**Review by Bachibouzouk — APPROVED** [review]
Thanks @srhbrnds

**Pull request merged**